### ISSUE TITLE
Shortcuts won't open emoji selector on wayland

### ISSUE THREAD

**Issue by putzwasser** [FelipeFTN/Emoji-Copy#27]
## Bug Report

**Current Behavior**
Emoji copy keybind isn't working on arch linux + wayland. Neither <Super> + . nor <Alt> + . opens the selector.

**Expected behavior/code**
Emoji copy gets opened on pressing keybind.

**Environment**
- emoji-copy version(s): 14.0
- npm version: 🤷
- OS: Arch Linux  + Wayland

**Possible Solution**

**Additional context/Screenshots**
I don't know if wayland might cause the issue here.

**Comment by FelipeFTN** [comment]
Hello!! Thanks for reporting this issue!
Just to clarify, are you pressing the right keys? In order to open emoji copy, you need to press **Super** and **.** 🌠
The **Super** key is the Windows key in most keyboards.
Please, let me know if it works! 💯

**Comment by putzwasser** [comment]
Edited the text to clarify. GitHub swallowed \<Super\> and \<Alt\>.

I mean, obviously, I pressed the right buttons. I guess, someone tech-aware enough to open an issue on GitHub is able to press the right buttons :wink: .

I get, however, that the text was confusing before the edit :sweat_smile: 

Maybe you could look into that; now that this is out of the way :smile:

**Comment by NatVIII** [comment]
Howdy, do you have any extensions installed? If so, can you list those out? There may be a conflict there, it might be worth toggling extensions on and off to see if any conflicts exist. Perhaps one of them is blocking this extension somehow? I say this because I run wayland and the keybinds work perfectly, it's actually the reason I started using this extension. Happy Holidays 🎄🧧

**Comment by FelipeFTN** [comment]
Any update, @putzwasser? :eyes:

**Comment by putzwasser** [comment]
Nope, unfortunately, not. Still doesn't work. Tired to disable all plugins, but it did not help…

**Comment by putzwasser** [comment]
@FelipeFTN Okay, I managed to get it work partially.

`<CTRL>.` Works. Don't know why neither `<Super>.` nor `<Alt>.` works.

Anyway, it just works on some apps.

It doesn't work on
- Brave
- Obsidian
- VSCode
- Libreoffice

It works on
- gedit

Does your app produce logs somewhere? I'd like to take a look into that.

**Comment by FelipeFTN** [comment]
**Hello friend, @putzwasser!!** 👋 

Can you check if this is still happening to you?
I just tested here and seems to be working just fine!

I'm reviewing and solving some issues in the project!! 💯 
Please let me know about it!
